unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldCommon/ToolTipController.cs

```diff
@@ -13,11 +13,19 @@ public class ToolTipController : MonoBehaviour
 
     Coroutine changeAlphaCor;
 
+
+    private void OnDestroy()
+    {
+        if (changeAlphaCor != null)
```

Comment:
```suggestion
        if (changeAlphaCoroutine != null)
```

Please don't use abbreviations unless its strictly needed, it adds confusion to code legibility.